doc/onboarding.md

```diff
@@ -0,0 +1,88 @@
+# Numberscope onboarding
+
+## Accounts, etc.
+
+1. If you don't have an account, sign up for [GitHub](https://github.com).
+2. Make sure you've been added to the Numberscope GitHub organization.
+3. Make sure you've been added to the chat app (Slack or Zulip).
+
+## Setting up your computer for development
+
+1. If you don't have one, install a good text editor. Use whatever you want!
+   [Visual Studio Code](https://code.visualstudio.com/) is popular, and it
+   works well with our code base. If you want to use Visual Studio Code,
+   install the following extensions:
+
+    - [ ] Code Spell Checker
+    - [ ] TypeScript Vue Plugin (Volar)
+    - [ ] Vue Language Features (Volar)
+
```

Comment:
My guess is these appear as checkboxes in github-flavored markdown. In mkdocs (python-markdown) they are just plain  pairs of brackets after bullet points. Please either modify the markdown based on that difference, or let me know that you don't care about there just being a pair of brackets on each bullet item on the web version of the docs.

Reply:
Changed to regular list in https://github.com/numberscope/frontscope/pull/148/commits/59fd25623bc1f78bae3e0e097eb8fbecb53e6926.